### ISSUE TITLE
Custom file provider is not applied when using globing pattern

### ISSUE THREAD

**Issue by AlexBar** [ligershark/WebOptimizer#63]
**Steps**: Add files via globing pattern in ```services.AddWebOptimizer``` and specify custom file provider  via ```UseFileProvider/UseContentRoot```:
```csharp
// ...
services.AddWebOptimizer(pipeline =>
{
   //  ...
    pipeline.AddFiles("application/javascript; charset=UTF-8", "scripts/**/*.js")
                 .MinifyJavaScript()
                 .ForEach(asset => asset.UseContentRoot());
   //  ...
}
```
**Actual result**: An exception is thrown
```csharp
FileNotFoundException: No files found matching "/scripts/list.js" exist in "D:\Dev\TestProject\wwwroot\"
WebOptimizer.Asset.ExpandGlobs(IAsset asset, IHostingEnvironment env)
```

**Expected result**: Processed with no errors. File should be searched in project root.

**Root cause**: ```asset.Items``` collection is not copied into newly created asset.

**PS**: it works only if script is exlicitly specified without using globbing pattern.

**Comment by AlexBar** [comment]
Fixed in #64

**Comment by AlexBar** [comment]
madskristensen, can you update nuget package?
